app/views/canvas/_edit.html.erb

```diff
@@ -71,23 +71,21 @@
   </span>
 </div>
 <div id="diagram-container">
-  <div id="diagram" class="diagram">
-    <% my_modules.each do |my_module| %>
-      <%= render partial: "canvas/edit/my_module", locals: {experiment: @experiment, my_module: my_module} %>
-    <% end %>
+  <% if can_manage_experiment?(@experiment) %>
+    <div id="diagram" class="diagram">
+      <% my_modules.each do |my_module| %>
+        <%= render partial: "canvas/edit/my_module", locals: {experiment: @experiment, my_module: my_module} %>
```

Comment:
add space here { experiment: @experiment, my_module: my_module }

Reply:
Oh you OCD, you!